udata_hydra/analysis/csv.py

```diff
@@ -74,7 +75,7 @@
 minio_client = MinIOClient()
 
 
-async def notify_udata(check_id: str, table_name: str) -> None:
+async def notify_udata(check_id: int, table_name: str) -> None:
```

Comment:
Wait, it was typed as `str`, and now it's typed as `int`, but there's no change related to that type change anywhere in the code?
I mean, was it just badly typed before, and went undetected? If so, would we want some kind of automatic check/validation to prevent this?

If the typings are meant to help us, but they could be wrong, how useful are they really?

I think this should be adressed in another PR or discussed during a meeting, what do you think?